docs/it/userinfo_endpoint.rst

```diff
@@ -27,11 +33,15 @@ Lo UserInfo Endpoint DEVE supportare l'uso del solo metodo HTTP GET :rfc:`2616`
 Response
 ++++++++
 
-La response dello UserInfo Endpoint DEVE specificare nel "Content-Type" il valore "application/jwt".
+.. admonition:: |spid-icon|
+
+  La response dello UserInfo Endpoint DEVE specificare nel "Content-Type" il valore "application/jwt".
 
 Il contenuto del corpo della Response DEVE essere un `JWT firmato e cifrato. <https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse>`_
 
-Lo UserInfo Endpoint restituisce i claim autorizzati nella Authentication Request.
+L'header JOSE DEVE contenere il parametro *cty* (Content Type) valorizzato con *JWT* (vedi :rfc:`7515#section-4`).
```

Comment:
qui il testo si ripete, 

il ref è questo
https://datatracker.ietf.org/doc/html/rfc7519#section-5.2

Reply:
fixed in https://github.com/italia/spid-cie-oidc-docs/pull/55/commits/8ec33eca96a7075ce9c222eb16e07b2fc6820176